SharkCage/CageManager/CageManager.cpp

```diff
@@ -539,3 +547,51 @@ std::optional<std::wstring> generateUuid() {
 	::RpcStringFree(&uuid_str);
 	return uuid_stl;
 }
+
+void quitChildProcesses(std::wstring process_binary_name) 
+{
+	DWORD my_pid = GetCurrentProcessId();
+	HANDLE snapshot_handle = CreateToolhelp32Snapshot(TH32CS_SNAPPROCESS,0);
```

Comment:
almost the same thing (iterating over processes with `Toolhelp32Snapshot``is already done in `CageManager::ProcessRunning(...)`, should we combine them?

Reply:
I am not sure about that... the only thing it does similarly, is the walk through the snaphshot... maybe we could do something crazy like take a function pointer as an argument and execute that function of every process of given name, but it seems overkill to me

Reply:
alright, let's keep it as is